.github/dependabot.yml

```diff
@@ -0,0 +1,42 @@
+version: 2
+enable-beta-ecosystems: true
+updates:
+- package-ecosystem: "docker"
+  directory: "/"
+  schedule:
+    interval: weekly
+  labels:
+    - "area/dependency"
+    - "release-note-none"
+    - "ok-to-test"
+    - "kind/cleanup"
+- package-ecosystem: "docker"
+  directory: "/cmd/webhook"
```

Comment:
Is this only "/cmd/webhook" or "/cmd/webhook*"?

Reply:
`cmd/webhook` should be fine here for Dockerfile scanning.